Penteract/Assets/Animations/Onimaru/StateMachine.stma

```diff
@@ -1,783 +1,2096 @@
+        443946419548588007,
+        13048874284171053028,
+        1206606529055351865,
+        4581045705644746094,
+        1440176599579779755,
+        2055027004229309011,
+        18390193090283511302,
+        5627254768085510119,
+        9086944810664241375,
+        13345940188439716691
     ],
     "States": [
         {
-            "Id": 4238491720123368409,
-            "Name": "RunBackwardRight",
-            "ClipId": 22413389010585387
-        },
-        {
-            "Id": 14622544802797745290,
+            "Id": 5764226279103904284,
             "Name": "Death",
-            "ClipId": 15962635220699174016
+            "ClipId": 7088583440786574377
         },
         {
-            "Id": 10600847267773113542,
-            "Name": "Idle",
-            "ClipId": 13702786571220342475
+            "Id": 13467888181552141058,
+            "Name": "EnergyBlast",
+            "ClipId": 17173081407878853217
         },
         {
-            "Id": 17645799573883922878,
+            "Id": 10595126271212778276,
             "Name": "RunForward",
-            "ClipId": 12301787421839207752
+            "ClipId": 9898232257833817295
         },
         {
-            "Id": 7470896876223194313,
-            "Name": "RunBackward",
-            "ClipId": 476354942140974123
+            "Id": 873641734253888914,
+            "Name": "Idle",
+            "ClipId": 13327903619887085228
         },
         {
-            "Id": 15734760037381800401,
-            "Name": "RunForwardLeft",
-            "ClipId": 7986897975056373172
+            "Id": 12470113208356225937,
+            "Name": "RunBackward",
+            "ClipId": 10637617280011635050
         },
         {
-            "Id": 18372065666961549589,
+            "Id": 17783557562039682859,
             "Name": "RunBackwardLeft",
-            "ClipId": 5580563623720458094
+            "ClipId": 4511330797560556630
         },
         {
-            "Id": 3427636900428404341,
+            "Id": 5813470502174396486,
+            "Name": "RunBackwardRight",
+            "ClipId": 7206609954069743860
+        },
+        {
+            "Id": 16897345640417184736,
+            "Name": "RunForwardLeft",
+            "ClipId": 443946419548588007
+        },
+        {
+            "Id": 10903485862385788043,
             "Name": "RunForwardRight",
-            "ClipId": 13233531638817105526
+            "ClipId": 13048874284171053028
         },
         {
-            "Id": 6051888174089562843,
+            "Id": 17610375932280740417,
             "Name": "RunLeft",
-            "ClipId": 12944952509532699935
+            "ClipId": 1206606529055351865
         },
         {
-            "Id": 7370649363945033478,
+            "Id": 1968081400917878446,
             "Name": "RunRight",
-            "ClipId": 11898288541297174563
+            "ClipId": 4581045705644746094
+        },
+        {
+            "Id": 11307525191858896573,
+            "Name": "Shield",
```

Comment:
State machine principal doesn't has to have states of the secondary state machine. 
So need to delete states Shield, Shooting and ShootingShield, and also all his transitions